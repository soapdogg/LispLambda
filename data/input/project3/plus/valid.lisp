(PLUS 8 4)
(PLUS (PLUS 4 3) (PLUS (PLUS 1 2) 4))
(PLUS (COND ((GREATER 3 45) 1) (T 2)) 2854)
(PLUS 1 2)
(PLUS (PLUS 3 5) (TIMES 4 4))
(PLUS (PLUS 3 5) (CAR (QUOTE (7 8))))