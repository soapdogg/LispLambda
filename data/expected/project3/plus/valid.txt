12
14
2856
3
24
15
